public/app/plugins/datasource/postgres/partials/config.html

```diff
@@ -41,6 +41,17 @@ <h3 class="page-heading">PostgreSQL Connection</h3>
 <h3 class="page-heading">PostgreSQL details</h3>
 
 <div class="gf-form-group">
+	<div class="gf-form">
+		<span class="gf-form-label width-9">
+			Version
+			<info-popover mode="right-normal" position="top center">
+				This option controls what functions are available in the PostgreSQL query builder.
+			</info-popover>
+		</span>
+		<span class="gf-form-select-wrapper">
+			<select class="gf-form-input gf-size-auto" ng-model="ctrl.current.jsonData.postgresVersion" ng-options="f.value as f.name for f in ctrl.postgresVersions"></select>
+		</span>
+	</div>
 	<div class="gf-form">
 		<gf-form-switch class="gf-form" label="TimescaleDB" tooltip="Use TimescaleDB features (e.g., time_bucket) in Grafana" label-class="width-9" checked="ctrl.current.jsonData.timescaledb" switch-class="max-width-6"></gf-form-switch>
```

Comment:
For a new user that don't know what TimescaleDB is I think the description could be better. 

Maybe use similar approach to http datasources which have a Help button, see [code](https://github.com/grafana/grafana/blob/master/public/app/features/plugins/partials/ds_http_settings.html#L32). 

A short description with a link to TimescaleDB github repository would be nice I think.